akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/scaladsl/EventProducerPushDestination.scala

```diff
@@ -45,6 +47,7 @@ object EventProducerPushDestination {
       (_, _) => Transformation.empty,
       None,
       immutable.Seq.empty,
+      immutable.Seq.empty,
```

Comment:
should we even make it a mandatory parameter of `apply` so that the user notice that they may need to provide it, given that it's our recommendation. I think it's mandatory in `GrpcReadJournal`.

Reply:
Yes, good point, let's do that.